packages/anthropic/src/convert-to-anthropic-messages-prompt.test.ts

```diff
@@ -213,7 +213,7 @@ describe('user messages', () => {
             content: [
               {
                 type: 'file',
-                data: 'base64data',
+                data: 'data:text/plain;base64,SGVsbG8gV29ybGQ=',
```

Comment:
is this the right format here? file content parts do no contain string like that afaik